DSE_6.0_Release_Notes.md

```diff
@@ -5,6 +5,36 @@ DSE 6.0.x is compatible with Apache Cassandra&trade; 3.11 and adds additional pr
 Please consider upgrading to [DSE 6.8](./DSE_6.8_Release_Notes.md) for our latest features and patches.
 
 
+# Release notes for 6.0.18
+27 May 2022
```

Comment:
This date is probably incorrect.
Should it be changes somewhere?
And if so... what steps from the checklist should be repeated?

Reply:
You can changed it directly in the PR